src/message/messageActionSheet.js

```diff
@@ -427,12 +432,17 @@ export const showTopicActionSheet = ({
     unreadStreams: UnreadStreamsState,
     ...
   }>,
-  stream: string,
+  streamId?: number,
```

Comment:
This should also not be optional. I suspect it may be simpler to keep this taking a `stream: string` for now, so that we can centralize the error handling (since currently, both of the callers get the stream id by searching the subscriptions), but I don't have a strong feeling on that.

Reply:
using `streamId` does cause me to repeat the error handling code at the calling site. But If we use `stream` we would have a mixture of two types of ActionSheet (one wanting stream and one wanting streamId) since there are many places where I can easily use `StreamId` (without searching). If we don't do this mixup we would be promoting the use of `stream` where we could easily have used `streamId` which may make using `showTopicActionSheet` difficult in some new places or it may make the refactor we would eventually do, difficult.

This is not a very convincing argument by me but I am still using `streamId`.